virtcontainers/utils/utils.go

```diff
@@ -105,6 +105,15 @@ func WriteToFile(path string, data []byte) error {
 	return nil
 }
 
+//CalculatemCPUs converts CPU quota and period to milli-CPUs
+func CalculatemCPUs(quota int64, period uint64) uint {
+	if quota != 0 && period != 0 {
+		return uint((uint64(quota) * 100) / period)
```

Comment:
As an int64, `quota` can be -1 to indicate unconstrainted resources. And it's better to also check int overflow as an utils package.

Reply:
Other parts LGTM

Reply:
Could you rename this to something like `CalculateMilliCPUs()` - the "`m`" in `CalculatemCPUs()` isn't immediately obvious so we may as well expand it for clarity.

Reply:
ACK - renamed, and handling the -1 case.